index.md

```diff
@@ -18,6 +18,12 @@ layout: default
 - High Availability and Scalability with Akka Cluster
 - Provides responsiveness by making the application stateful, while providing fast failover with guaranteed consistency using Raft
 
+
+<object type="image/svg+xml" data="https://cdn.jsdelivr.net/gh/lerna-stack/lerna-stack.github.io@master/img/benchmarkscore_en.drawio.svg"></object>
+
+
+* See availability and throughput of Lerna for more information on prerequisites and scores for benchmark testing(coming soon)
```

Comment:
@yugolf 
[IMO] I think we could write it more simply.
(e.g. Benchmark test report of Lerna available here)

Reply:
I changed the sentences as you pointed out.